lib/Rest/GeneralResponse.h

```diff
@@ -158,7 +158,7 @@ class GeneralResponse {
   bool generateBody() const { return _generateBody; };  // used for head
```

Comment:
OK then, remove the ';' also at end of  line 107

Reply:
yes, will do!

Reply:
ok, done.